.drone.yml

```diff
@@ -254,6 +254,27 @@ pipeline:
     when:
       event: [ push, tag ]
 
+  gpg-sign:
+    image: plugins/gpgsign:1
+    pull: true
+    secrets: [ gpgsign_key, gpgsign_passphrase ]
+    detach_sign: true
+    files:
+      - dist/release/*-amd64
```

Comment:
should we sign the compressed files too?

Reply:
Since they should be extract to be usefull ? It is possible but if it un-needed but if someone want it we can.

Reply:
I think it might be a good idea because if perhaps someone replaces the compressed file with perhaps a zip bomb. Signing the compressed file means they can check it before they extract.

Reply:
done

Reply:
Only the xz files are getting uploaded, right? Than only sign *.xz

Reply:
Both are uploaded, compressed and non-compressed binary. We are signing both.

Reply:
Maybe we should also add an exclude pattern to the plugin, than we can simply add

```
    files:
      - dist/release/*
    excludes:
      - dist/release/*.sha265
```

Reply:
Created a pr for the plugin, you can see the full example at https://github.com/drone-plugins/drone-gpgsign/pull/2#issuecomment-399097791

Reply:
Should we wait for that PR to be merged into the plugin, or just use the proposed solution and change the to the new excludes config for 1.6/1.5.1?

Reply:
First there have to be the secrets anyway... And than it's up to you, I guess the plugin gets merged soon.

Reply:
@daviian This could be done later to not block 1.5.0. I will PR the change after but it only offer simplicity to the build process and won't even need backport.

Reply:
@sapk I'd prefer that.

@go-gitea/owners Have you already added the gpg keys to the drone secret? :)

Reply:
I think we should wait https://github.com/drone-plugins/drone-gpgsign/pull/2#issuecomment-399097791 merge?

Reply:
That's only a cosmetical improvement and can be added anytime. I don't see a need to delay this milestone any further.

Reply:
I can merge the gpgsign plugin, but to tag a release I have to be home

Reply:
As long as we can merge and release RC today I'm fine

Reply:
Merged gpgsign PR and triggered a release => https://beta.drone.io/drone-plugins/drone-gpgsign/11

Reply:
@lunny @tboerger done.